curator-recipes/src/test/java/org/apache/curator/framework/recipes/leader/TestLeaderSelectorEdges.java

```diff
@@ -98,15 +97,13 @@ public void flappingTest() throws Exception {
         } finally {
             try {
                 leaderSelector1.close();
-            } catch (IllegalStateException e) {
-                fail(e.getMessage());
```

Comment:
This shadow the real cause.

See https://ge.apache.org/s/rcubepxch5ibu/tests/goal/org.apache.curator:curator-recipes:surefire:test@default-test/details/org.apache.curator.framework.recipes.leader.TestLeaderSelectorEdges/flappingTest?top-execution=1